tests/fsapt-diff1/input.dat

```diff
@@ -1,71 +0,0 @@
-#! This test case shows an example of running and analyzing a difference
-#! F-SAPT0/jun-cc-pvdz procedure for phenol dimer from the S22 database.
-
```

Comment:
in what way isn't this demo working as a plain test case? it seems like it has some good guidance that I wouldn't like to lose if it doesn't live somewhere else, too.

Reply:
`fsapt-diff1/input.dat` is just a copy of `fsapt1/input.dat` (though looks like needs an updated reference value, cf. https://github.com/psi4/psi4/blame/master/tests/fsapt1/input.dat#L53). I'll add this back since it looks like the reference output (`fsaptD-ref`) is useful, but it might be nice to actually add a test for this data at some point so we know it is actually reproducible (it does seem to be currently).

Reply:
neat, the nre was all it needed! I went from having 100 files left to review to having one loc change :-)

agree fsapt-diff1 could use some attention, but that can be future pass. thanks for patching it up.